opm/output/eclipse/RestartIO.cpp

```diff
@@ -509,8 +541,10 @@ void save(const std::string& filename,
           data::Wells wells,
           const EclipseState& es,
           const EclipseGrid& grid,
+          std::map<std::string, std::vector<double>> extra_data,
```

Comment:
I looked at this and found no errors. There are a few minor things I would change, though, that I did not notice before. The `wells` and `extra_data` arguments should be passed as const references (unlike `cells` they are not modified inside this function). Also, I'd put the `reserved_keys` inside the checking scope below. Not that it should matter, but could there be a problem with multithreaded calls to this function, because of the nontrivial static variable (perhaps someone with more knowledge about this can enlighten me)? I think removing the `static` optimization would not harm performance in any case since this is after all a function that writes to disk.

I cannot reproduce, but on Travis the error is triggered in the function that uses the default parameter. Perhaps removing that default parameter could shed some light on it?